sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -258,9 +263,9 @@ internal class RadarLocationManager(
             RadarSettings.setShouldListenToServerTrackingOptions(context, true)
         } else {
             // Fallback
-            logger.d("Removing server tracking options | options = ${Radar.getTrackingOptions()}")
             RadarSettings.removeRemoteTrackingOptions(context)
             RadarSettings.setShouldListenToServerTrackingOptions(context, false)
+            logger.d("Removing server tracking options | options = ${Radar.getTrackingOptions()}")
```

Comment:
Yes - the significance would be to fix the logging, such that it actually logs the fallback value (not the current RTO). I kept it as "removing" since that's the standard we have for any logging - verbs are always present tense.